plugins/snapshot_api_objects.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
+  name: snapshot_api_objects
+spec:
+  # 'version' is a valid semantic version string (see semver.org). Note the prefix 'v' is required.
+  version: "v0.0.5"
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: https://github.com/oguzhan-yilmaz/kubectl-snapshot-api-objects
+  # 'shortDescription' explains what the plugin does in only a few words
+  shortDescription: "Snapshot Kubernetes API Objects"
+  description: |
+    Snapshot all Kubernetes Objects and save them as yaml files in .tar.gz archive.
```

Comment:
Does kubectl get-all  plugin let us do this? If so, it might be good to not create another plugin and instead compose them.